posthog/settings/object_storage.py

```diff
@@ -6,7 +6,7 @@
 from posthog.utils import str_to_bool
 
 if TEST or DEBUG:
-    OBJECT_STORAGE_ENDPOINT = os.getenv("OBJECT_STORAGE_ENDPOINT", "http://localhost:19000")
+    OBJECT_STORAGE_ENDPOINT = os.getenv("OBJECT_STORAGE_ENDPOINT", "http://objectstorage:19000")
```

Comment:
@timgl Did you mean to make this change? It causes some of the experiment tests to fail for me locally because I didn't have `objectstorage` mapped to `127.0.0.1`.

Reply:
+1